libc/cmake/modules/LLVMLibCObjectRules.cmake

```diff
@@ -43,6 +43,10 @@ function(_get_common_compile_options output_var flags)
     list(APPEND compile_options "-fno-rtti")
     list(APPEND compile_options "-Wall")
     list(APPEND compile_options "-Wextra")
+    # -DLLVM_LIBC_WNO_ERROR=ON if you can't build cleanly with -Werror.
+    if(NOT LLVM_LIBC_WNO_ERROR)
```

Comment:
Can you change the flag to `LIBC_WNO_ERROR`?  We plan to convert all the current public flags that start with `LLVM_LIBC_*` to `LIBC_*` so that they work well with runtime builds.

Reply:
done in https://github.com/llvm/llvm-project/pull/73966/commits/1fa2e4df97a85dc6dbede03d3e4a1cfbd65228db PTAL